doc/rest-api/Client-frontend.md

```diff
@@ -55,6 +55,40 @@ By default the REST API is exposed on port 8089 but this can be changed to whate
 For more details about possible `ejabberd_cowboy` configuration parameters please
 see the relevant documentation in the [Listener modules](../advanced-configuration/Listener-modules.md#http-based-services-bosh-websocket-rest-ejabberd_cowboy).
 
+
+## Smack library support
+REST API can fetch messages for [Smack](https://github.com/igniterealtime/Smack/blob/master/documentation/extensions/properties.md#stanza-properties) Stanza Properties.
+
+For example if we have properties in the stanza like:
+  ```
+      <message xml:lang='en' to='alice@localhost' id='123' type='chat'>
+        <body xml:lang='en_US'>Hi!</body>
+        <properties xmlns="http://www.jivesoftware.com/xmlns/xmpp/properties"
+            <property>
+                <name>some_number</name>
+                <value type='integer'>123</value>
+            <property>
+            <property>
+                <name>some_string</name>
+                <value type='string'>abc</value>
+            <property>
+        </properties>
+      </message>
+  ```
+then in the final json message these properties will be converted to json map without tag names:
+```
+    {   "to": "alice@localhost",
+        "timestamp": 1531329049949,
+        "id": "123",
+        "from": "bob@localhost",
+        "body": "Hi!",
+        "properties":{
+            "some_number":123,
```

Comment:
Looking at the code and tests I'm not sure the number will be returned as number. It seems to be (mostly based on the `convert_prop_child`) that the number will be encoded as a JSON string. I don't think this would be expected.

Reply:
@michalwski, you are right.  Now it's fixed.